src/pages/ProductPage/ProductPage.jsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
+import axios from "axios";
 import "./ProductPage.css";
-import { Aside, ProductGrid, Loader } from "../../components/index";
+import { Aside, Loader, ProductGrid } from "../../components/index";
+import { useEffect, useState } from "react";
 export function ProductPage() {
+  const [loader, setLoader] = useState(true);
+  const [products, setProducts] = useState([]);
+
+  async function getProducts() {
+    setLoader(true);
+    const { data } = await axios.get("./api/products");
+    setLoader(false);
+    setProducts(data.products);
+  }
```

Comment:
You should use try catch for the server call.

Reply:
Yes I will add that